src/extensions/default/bramble/lib/LinkManagerRemote.js

```diff
@@ -0,0 +1,28 @@
+/*jslint vars: true, plusplus: true, devel: true, nomen: true, indent: 4, forin: true, maxerr: 50, regexp: true, bitwise: true */
+/* global addEventListener, window */
+(function() {
+    "use strict";
+
+    function handleClick(e) {
+        var url = e.target.getAttribute("href");
+
+        // For local paths vs. absolute URLs, try to open the right file
+        if(!(/\:?\/\//.test(url)) && window._Brackets_LiveDev_Transport) {
+            window._Brackets_LiveDev_Transport.send("bramble-navigate:" + url);
+        } else {
+            window.open(url, "_blank");
+        }
+
+        return false;
```

Comment:
This will do what you want with `e.stopPropagation()` and `e.preventDefault()` (i.e., `return false` does both).